openedx/core/djangoapps/user_api/accounts/serializers.py

```diff
@@ -88,6 +88,7 @@ def to_representation(self, user):
             "level_of_education": AccountLegacyProfileSerializer.convert_empty_to_None(profile.level_of_education),
             "mailing_address": profile.mailing_address,
             "requires_parental_consent": profile.requires_parental_consent(),
+            "account_privacy": UserPreference.get_value(user, 'account_privacy'),
```

Comment:
@jcdyer Since the user preferences model is a generic key-value-store for all of a user's preferences, we don't know what type of data (large or small) we'll eventually store there.  The optimization you suggest is under the assumption that we'll have few user preferences and eventually multiple preference queries in this code.  Since this is unknown at this time, I'd prefer not to over-optimize it and revisit it when we optimize the Teams API.